### PR TITLE
Added PhpdocAnnotationWithoutDotFixer

### PR DESCRIPTION
**Pull request by keradus**
ref https://github.com/symfony/symfony/pull/19198

### CODE REVIEW

**Comment by fabpot** [comment]
LGTM

**Comment by SpacePossum** [comment]
can we ditch other stop characters as well?  (`'.。!?¡¿！？'`)

**Comment by keradus** [comment]
https://github.com/symfony/symfony/pull/19198#issuecomment-228971663

**Comment by GrahamCampbell** [comment]
You should also lower case the first letter too then, because it's not a sentence.

**Comment by SpacePossum** [comment]
>You should also lower case the first letter too then, because it's not a sentence.

not all people write English only, so the rule doesn't always apply.

**Comment by GrahamCampbell** [comment]
What about those people who, like symfony?

**Comment by SpacePossum** [comment]
>What about those people who, like symfony?

the fixer might be used by other people than symfony people

**Comment by rob006** [comment]
First word might be the name of class, constant, acronym, etc., you can not safely change case of the first letter.

**Comment by GrahamCampbell** [comment]
Don't make the first word that then. I never do that because of that that exact issue. Allow me to have correctly formatted sentences.

**Comment by SpacePossum** [comment]
you can make an issue out of every new fixer and/or configuration option you don't want, but since no one forces you to use it, what is the point?

**Comment by keradus** [comment]
@GrahamCampbell if you want to put sentence to param description it is great ! Don't use the fixer that force you to not have it ;)

File: Symfony/CS/Fixer/Contrib/Php4ConstructorFixer.php [hunk 1]
```diff
@@ -315,7 +315,7 @@ private function getWrapperMethodSequence(Tokens $tokens, $method, $startIndex,
      *     - endIndex (int): The index of the function/method end.
      *     - bodyIndex (int): The index of the function/method body.
      *     - modifiers (array): The modifiers as array keys and their index as
-     *       the values, e.g. array(T_PUBLIC => 10).
+     *       the values, e.g. array(T_PUBLIC => 10)
```
Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Contrib/Php4ConstructorFixer.php]: This removal is invalid

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Contrib/Php4ConstructorFixer.php]: we should only remove on singleline descriptions, since this one has multiple sentences

Comment (keradus) [inline Symfony/CS/Fixer/Contrib/Php4ConstructorFixer.php]: the thing is that due to Symfony standards you shouldn't have sentence here in the first places.
(btw, single line could have multiple sentences as well, not connected)

Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Contrib/Php4ConstructorFixer.php]: Yeh. I would suggest removing the last line only if there was not a dot earlier on, but that wouldn't be reliable.

**Pull request closed without merging**